localisation/00_zuwc_united_localisation.csv

```diff
@@ -1693,7 +1693,7 @@ EVTOPTA_WCFEL_1005;The Qiraji will devour them all!;;Die Qiraji werden sie alle
 EVTOPTB_WCFEL_1005;The ancient evil must remain in confinement!;;Das uralte B�se muss eingesperrt bleiben!;;;;;;;;;;;x
 burn_evil_beings;Burn Evil Beings;;Verbrennt das B�se;;;;;;;;;;;x
 burn_evil_beings_desc;Burn all unholy characters as evil beings or get a reason to execute them if they're rulers. They deserve to die!;;Verbrennt alle unheiligen Wesen. Sie verdienen nur den Tod!;;;;;;;;;;;x
-burn_evil_being_effect_tooltip;�Y[This.GetBestName]�! is sentenced to be burnt\n;;�Y[This.GetBestName]�! wird zum Tod durch Verbrennung verurteilt.;;;;;;;;;;;x
+burn_evil_being_effect_tooltip;�Y[This.GetBestName]�! is sentenced to be burnt;;�Y[This.GetBestName]�! wird zum Tod durch Verbrennung verurteilt.;;;;;;;;;;;x
```

Comment:
Not necessarily a correction, but burnt is just more often used as an adjective. "Burnt food"
```suggestion
burn_evil_being_effect_tooltip;�Y[This.GetBestName]�! is sentenced to be burned;;�Y[This.GetBestName]�! wird zum Tod durch Verbrennung verurteilt.;;;;;;;;;;;x
```